dll/win32/shell32/precomp.h

```diff
@@ -50,14 +51,8 @@
  * For versions < Vista+, redefine ShellMessageBoxW to ShellMessageBoxWrapW
  * (this is needed to avoid a linker error). On Vista+ onwards, shell32.ShellMessageBoxW
  * redirects to shlwapi.ShellMessageBoxW so the #define should not be needed.
```

Comment:
This comment too should be moved to `compat.h`.

Reply:
Moved.